popsycle/synthetic.py

```diff
@@ -1921,10 +1852,9 @@ def unique_events(event_table):
     Parameters
     ---------
     event_table : numpy array 
-        A table with all the events. There are 66 columns: 31 with info about
-        the source, 31 with the corresponding information about the lens, and
-        4 with info about theta_E, u, mu_rel, and tstep. The number of rows
-        corresponds to the number of events.
+        A table with all the events. The columns contain info about
+        the source, the lens, and about theta_E, u0, mu_rel, and t0.
+        The number of rows corresponds to the number of events.
```

Comment:
(Add comment about how number of columns with info about source and lens are the same)

Reply:
done